server/schema/schema.json

```diff
@@ -0,0 +1,69 @@
+"users": 

```

Comment:
Why do we have a schema.json containing all the other Json? Curious question btw, not a critic